prov/efa/src/rdm/efa_rdm_peer.h

```diff
@@ -24,6 +24,12 @@ OFI_DECL_RECVWIN_BUF(struct efa_rdm_pke*, efa_rdm_robuf, uint32_t);
  */
 #define EFA_RDM_PEER_HANDSHAKE_QUEUED      BIT_ULL(5)
 
+struct efa_rdm_peer_user_recv_qp
+{
+	uint32_t qpn;
+	uint32_t qkey;
+};
```

Comment:
Nit: I think I'd like the definition to be something more generic like `struct efa_qp_attr` defined in efa_base_ep.h. Then `efa_qp.{qp_num,qkey}` can be replaced with the same struct; making the (de)serialization a little simpler and future-proof. Just an idea.

Reply:
It's good point. I have made a solid mind whether we should name {qp_num, qkey} as efa_qp_attr because there may be some other attribute that peer doesn't need here. Can be addressed in a separate PR.